src/source/Signaling/LwsApiCalls.c

```diff
@@ -632,6 +639,10 @@ STATUS lwsCompleteSync(PLwsCallInfo pCallInfo)
     return retStatus;
 }
 
+BOOL isCallResultFailureRetryable(PCallInfo pCallInfo) {
```

Comment:
If there is no plan to expand on this then this can be a macro. 

More importantly, checking for the text within the response message is a very brittle architecture. There should be a proper contractual signal for a particular failure.

Reply:
I should provide some context so an upcoming PR after this will actually automatically correct for clock skew in the SDK and retry at which time this may be removed altogether its kind of here as an intermediate step right now.  aws sdks have auto clock skew correction that will be feature so we're adding it here as well to help customers who are having a hard time with ntp syncing their devices to make sure time is accurate.